docs/Management.md

```diff
@@ -253,3 +279,69 @@ $ forge script \
     -vvv \
     script/${SCRIBE_FLAVOUR}.s.sol:${SCRIBE_FLAVOUR}Script
 ```
+
+## Offboarding
+
+Offboarding a Scribe(Optimistic) instance simply means _Chronicle Protocol_ is not guaranteeing pokes anymore, ie the oracle is not being updated anymore.
+
+However, to ensure an offboarded Scribe(Optimistic) instance may not behave unexpectedly it needs to be deactivated. Furthermore, if the contract is a ScribeOptimistic instance ETH held by the contract may need to be rescued. If its certain the contract will never be used again it is recommened to kill it.
+
+### Deactivation
+
+Deactivating a Scribe(Optimistic) instance means its value is set to zero, leading all `read()` calls to revert/fail, no feeds are lifted, and `bar` is set to `255`.
+
+Note that one or more addresses still hold `auth` on the contract meaning the instance can be reactivated via `lift`-ing feeds and updating `bar` again.
+
+> [!IMPORTANT]
+>
+> Deactivation requires running two distinct `forge script` commands.
+>
+> It is of utmost importance to run both commands and NOT leave the Scribe(Optimistic) instance in an undefined state.
```

Comment:
not relevant to this PR, but if these two steps need to be run atomically, I think we should provide a single step that executes both, and promote that. i'm specifically talking about keeping these commands, and just having a step (could be via `make` even) that ensures both run

Reply:
Yeah was thinking along the same lines. However, would not like to introduce a `Makefile` just for a command we rarely execute and so far all the `docs/{Deployment, Management}` commands are `forge script`s.

Therefore, I exaggerated a bit...😛 Its actually not bad at all to only run the first step. The only thing missing is Scribe not having the zero value set, but otherwise no harm done.